### PR TITLE
Changed Upcoming Releases format.

### PR DESCRIPTION
**Pull request by russjones**

### CODE REVIEW

File: docs/pages/preview/upcoming-releases.mdx [hunk 1]
```diff
@@ -52,32 +49,40 @@ The key deliverables for Teleport Cloud in the next quarter:
...
+#### Patch Releases
 
-- Releases `11.0.0` and `12.0.0` are major releases. We publish 4 major releases
-  each year.  Read more about upgrades and compatibility
-  [here](../management/operations/upgrading.mdx).
-- Releases `11.1.0` are minor releases. They contain minor backwards-compatible
-  improvements and backports.
-- Versions like `11.0.1` are quick patches. They contain backwards-compatible
-  fixes.
+Patch releases contain small bug fixes and are always ready for production use.
```
Comment (xinding33) [inline docs/pages/preview/upcoming-releases.mdx:88]:
```suggestion
Patch releases contain small bug fixes and can typically be deployed directly to production.
```

File: docs/pages/preview/upcoming-releases.mdx [hunk 2]
```diff
@@ -52,32 +49,40 @@ The key deliverables for Teleport Cloud in the next quarter:
...
 
-- You can use alpha releases such as `12.0.0-alpha.1` for trying new features.
-  Things can break and new changes may not be backwards-compatible.
-- Teleport `beta` releases, such as `12.0.0-beta.2` are suitable for staging
-  environments.  We are unlikely to change the APIs while we are ironing out
-  bugs and UX glitches.
-- We mark release candidates as `12.0.1-rc.1` coding and bug fixes are finished.
-  The team is going through the manual test plan to find any regressions.
+Minor releases of Teleport typically contain smaller features and improvements.
+Minor releases can be deployed directly to production.
```
Comment (xinding33) [inline docs/pages/preview/upcoming-releases.mdx:81]: Minor releases can typically be deployed directly to production.

**Review by xinding33 — COMMENTED** [review]
@russjones Thanks for formalizing the changes we discussed. Besides the minor nits on language, I have two questions:
1. We discussed making it clear to users who are more conservative that they can use the latest stable release of the last major release of Teleport. I see that you did not include any mention of that here. After thinking about it, I actually think it's best to NOT include anything like that here. Just wanted to mention that in case you omitted on accident.
2. We also discussed the confusion that "Preview" causes. Is there appetite to clarify what Preview means? For what it's worth, we have internal documentation already: https://gravitational.slab.com/posts/product-preview-policy-i783t1lh.

**Comment by russjones** [comment]
@xinding33 Added `Preview Policy` section. Also yes, intentionally omitted, felt it was too confusing and not what we want to encourage customers to do anyway.

**Review by zmb3 — APPROVED** [review]

File: docs/pages/preview/upcoming-releases.mdx [hunk 3]
```diff
@@ -8,38 +8,35 @@ The Teleport team delivers a new major release roughly every 3 months.
...
+environment.
 
-### (Preview) Oracle support for Database Access
+_Delayed from Teleport 12.1.0._
+
+#### (Preview) Oracle support for Database Access
 
 Teleport will add the ability to connect to Oracle databases to Teleport
-Database Access.
+Database Access. Note, that Audit Log support will come in a later release.
```
Comment (jimbishopp) [inline docs/pages/preview/upcoming-releases.mdx:29]:
```suggestion
Database Access (Audit Log support will come in a later release).
```

**Review by jimbishopp — APPROVED** [review]

**Review by r0mant — APPROVED** [review]

**Pull request merged**